### ISSUE TITLE
I can't use it

### ISSUE THREAD

**Issue by xXAlexJaja** [magcius/noclip.website#36]
Whatever I do, I can't open webgl on this page. Even if it says that works, on this page says it doesn't. I enabled webgl, relaunched chrome, and still doesn't work.

**Comment by magcius** [comment]
Can you paste the full error message? What OS and GPU are you using? Can you paste a screenshot of http://webglreport.com/?v=2 in here?

**Comment by magcius** [comment]
Please reopen once you have answered the questions.

**Comment by xXAlexJaja** [comment]
My OS is a Windows 7 Home Premium 64 bits and my GPU is an Intel Core i3 CPU.
And here is the screenshots:
Error message when i enter noclip.website : https://cdn.discordapp.com/attachments/329591520942620672/554073501007478803/nocliperror1.PNG
Screenshot of webglreport : https://cdn.discordapp.com/attachments/329591520942620672/554073503750553632/nocliperror2.PNG (I checked the link that says ''get webgl'' and I also taked another screenshot [ https://cdn.discordapp.com/attachments/329591520942620672/554073504463716363/nocliperror4.PNG ])
And also, I enabled webgl on chrome ( https://cdn.discordapp.com/attachments/329591520942620672/554073503909806100/nocliperror3.PNG ) and try if this works, but it's not working yet.

Thank you for replying my issue.
